typescript/common-resolvers/src/types.ts

```diff
@@ -22,9 +22,9 @@ import type {
   WorkspaceCreateInput,
   WorkspaceWhereInput,
   WorkspaceWhereUniqueInput,
+  WorkspacePlan,
   WorkspaceType,
 } from "@labelflow/graphql-types";
-import { WorkspacePlan } from "@prisma/client";
```

Comment:
This one is quite subtle 😅 

Although `db` depends on `common-resolvers`, it was reusing `WorkspacePlan` from `@prisma/client`. Which, under the hood, equals to importing `db` from `common-resolvers` ([details](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client)).

It worked because of a combination of 2 facts:
* our Prisma files are currently generated in `node_modules/.prisma`.
* the `node_modules` of `common-resolvers` is shared with `db`.